packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/elasticsearch/ElasticsearchService.java

```diff
@@ -471,6 +471,34 @@ public <T, Partial> void update(final String index, final String id, final Parti
 		}
 	}
 
+	/**
+	 * Update a document from an index.
+	 *
+	 * @param index The index to remove the document from
+	 * @param id    The id of the document to remove
```

Comment:
Your JavaDoc is missing `partial` and `routing`.

Reply:
Done